builder/googlecompute/step_create_instance.go

```diff
@@ -16,9 +16,10 @@ type StepCreateInstance struct {
 	Debug bool
 }
 
-func (c *Config) createInstanceMetadata(sourceImage *Image, sshPublicKey string) (map[string]string, error) {
+func (c *Config) createInstanceMetadata(sourceImage *Image, sshPublicKey string) (map[string]string, *packer.MultiError) {
```

Comment:
```suggestion
func (c *Config) createInstanceMetadata(sourceImage *Image, sshPublicKey string) (map[string]string, error) {
```

here `*packer.MultiError` is also an `error` and I think returning an `error` here would be better as it makes code simple(r).

Watch out though as `var errs *packer.MultiError` is not a nil error. ex: https://play.golang.org/p/1FEdNz1dFhi